.github/workflows/verify-repo-tags.yml

```diff
@@ -0,0 +1,80 @@
+name: "Verify Repo Tags"
+
+on:
+  schedule:
+    - cron:  '0 7 * * *'
```

Comment:
I know the Trello card says "every day", but we should probably change this to Mon-Fri, to avoid Slack spam.